frontend/proxy.conf.js

```diff
@@ -85,7 +85,7 @@ if (configContent && configContent.BASE_MODULE == "liquid") {
     });
 } else {
     PROXY_CONFIG.push({
-        context: ['/resources/pools.json', '/resources/assets.json', '/resources/assets.minimal.json'],
+        context: ['/resources/pools.json', '/resources/assets.json', '/resources/assets.minimal.json', '/ressources/worldmap.json'],
```

Comment:
ressources -> resources spelling. But why fetch the worldmap json from mempool.space when using local dev? Shouldn't the worldmap json data just be accessed from the local resources folder.